.github/workflows/auto-comment.yml

```
@@ -0,0 +1,27 @@
+    steps:
+      - name: Create PR comment
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `@coderabbitai full review
+
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 21: did not find expected key

(syntax-check)